src/common/snippets/include/snippets/pass/tokenization.hpp

```diff
@@ -108,6 +114,9 @@ class SnippetsTokenization : public ov::pass::ModelPass {
         // Otherwise, it may be fused into Subgraph if possible
         // TODO [111813]: Remove please when the ticket 111813 is implemented
         bool mha_token_enable_transpose_on_output = true;
+        // True if dynamic MHA is supported by backend
```

Comment:
This comment is bit misleading, since the problem with dynamic MHA support is not on the backend's side. 
Also, could you please add that currently this flag should be enabled only for test purposes?

Reply:
Discussed offline: 
1. Added comment about test purposes
2. Renamed to `is_dynamic_mha_token_enabled` instead of `supported`